test/integration/056_column_type_tests/macros/test_is_type.sql

```diff
@@ -65,5 +64,5 @@
         {% endif %}
     {% endfor %}
     {% do log('bad columns: ' ~ bad_columns, info=True) %}
-    select {{ bad_columns | length }} as pass_fail
-{% endmacro %}
+    select * from unnest(string_to_array('{{ bad_columns|join(',') }}', ','))
```

Comment:
fancy! :)

This works well on Postgres. If we needed to run it on any database, I think we could do:
```sql
{% for bad_column in bad_columns %}
  select '{{ bad_column }}' as bad_column
  {{ 'union all' if not loop.last }}
{% endfor %}
```

Reply:
I totally agree! The piece here that I'm struggling to handle correctly is if there are no bad columns (`bad_columns == []`), in this case, no sql will be rendered.

Reply:
Ohh right! So this is perhaps a bit jankier, but perhaps something like:
```
{% for bad_column in bad_columns %}
  select '{{ bad_column }}' as bad_column union all
{% endfor %}
  select * from (select 1 limit 0)
```

If we only run this test on postgres, I like your answer better :)